forc/src/cli/commands/deploy.rs

```diff
@@ -8,12 +8,18 @@ pub struct Command {
     /// Path to the project, if not specified, current working directory will be used.
     #[structopt(short, long)]
     pub path: Option<String>,
+    /// Whether to compile using the IR pipeline.
+    #[structopt(long)]
+    pub use_ir: bool,
     /// Whether to compile to bytecode (false) or to print out the generated ASM (true).
     #[structopt(long)]
     pub print_finalized_asm: bool,
     /// Whether to compile to bytecode (false) or to print out the generated ASM (true).
     #[structopt(long)]
     pub print_intermediate_asm: bool,
+    /// Whether to compile to bytecode (false) or to print out the IR (true).
+    #[structopt(long)]
+    pub print_ir: bool,
```

Comment:
Is this really needed for deploy? If they want to see the IR they can just run build.

Reply:
The `print_ir` flags are symmetrical to the already existing `print_intermediate_asm` and `print_finalized_asm` flags.

If they're set then the compilation will stop at the end and print out the ASM or IR.  They're useful for anyone, not just us.

There's no way to inspect the IR by just running build.  The `--use-ir` flag turns the IR compilation pipeline on, the `--print-ir` flag implies it.  I didn't bother documenting this as the `--use-ir` flag is temporary, it'll go away when we switch to IR full time.

Reply:
That makes sense.